src/main/resources/data/brass_armory/recipes/warp_arrow.json

```diff
@@ -0,0 +1,23 @@
+{
+  "type": "minecraft:crafting_shaped",
+  "pattern": [
+    "X",
+    "#",
+    "Y"
+  ],
+  "key": {
+    "#": {
+      "item": "minecraft:stick"
+    },
+    "X": {
+      "item": "minecraft:obsidian"
```

Comment:
Not an ender pearl?

Reply:
ender pearl makes it hard to craft, and most people would end up just using ender pearls instead. Idk it's a hard one to balance but this still means you have to be decently far in the game before you can use them (Diamond Pick) whereas ender pearls could be gotten very early on. 

Also, 1 ender pearl = 4 arrows or even 2 arrows feels like too many, whereas 4 or maybe 2  from 1 obsidian instead makes sense.

Reply:
But you can shoot the arrow way further right? Which makes it pretty useful. And then only craft one? It just feels weird to me that obsidian would make you teleport. I'm gonna think about it some more, maybe a different item

Reply:
Hmm good point, obsidian does feel weird for the teleporting. It's definitely gonna be a hard one to balance, let me know what you come up with. 

I guess the question is (for whether to use an ender pearl for crafting) do we want it to just straight up be a replacement for Ender Pearls late game? Or do we want the player to still have incentives for using both?

Reply:
For now I'm going to merge this, if you have an idea about a different crafting item either change it or let me know @Xratedjunior